core-bundle/src/Resources/contao/classes/Backend.php

```diff
@@ -537,19 +539,17 @@ protected function getBackendModule($module, PickerInterface $picker = null)
 							$trail[] = ' <span>' . $GLOBALS['TL_LANG']['MOD'][$table] . '</span>';
 						}
 
+						$strUrl = $container->get('router')->generate('contao_backend', array
+						(
+							'do' => $container->get('request_stack')->getCurrentRequest()->query->get('do'),
+							'table' => $table,
```

Comment:
how come you're using `$table` here but the loop uses `$ptable`? looks like this won't work with more than one level?

Reply:
Hmm yeah depends on the mode I guess. When editing a content element, `tl_content` is actually correct 🤔

Reply:
For the article case, it is correct. I don't even know if we have anywhere where we go deeper than that?

Reply:
`$table` is actually correct :) If you edit the content element of an event, it will look like this now:

<img width="985" alt="Bildschirmfoto 2021-11-26 um 16 13 23" src="https://user-images.githubusercontent.com/481937/143601088-328c0663-8795-4f39-9772-7f12605b960e.png">

* `Contao North-Day` links to `contao?do=calendar&table=tl_content&id=6.`
* `Contao Public Example Events` links to `contao?do=calendar&table=tl_calendar_events&id=3`